spec/controllers/account/payments_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe Account::PaymentsController, type: :controller do
+  let!(:utility_provider) { create(:billing_contract) }
```

Comment:
Do you need to create  provider, if you dont check anything related to it in specs?